src/components/Profile/components/PersonalInfoList/index.jsx

```diff
@@ -87,12 +87,16 @@ const PersonalInfoList = ({ myProf, profile, isOnline, createSnackbar }) => {
 
   // FacStaff spouses are private for private users
   const isSpousePrivate = isFacStaff && keepPrivate && profile.SpouseName !== PRIVATE_INFO;
-
   useEffect(() => {
     async function loadMailboxCombination() {
```

Comment:
Can you rename this function, since it's no longer just loading the mailbox combo?